config/initializers/warden.rb

```diff
@@ -0,0 +1,24 @@
+require 'open_project/authentication'
+
+# Strategies provided by OpenProject:
+require 'open_project/authentication/strategies/warden/basic_auth_failure'
+require 'open_project/authentication/strategies/warden/global_basic_auth'
+require 'open_project/authentication/strategies/warden/user_basic_auth'
+require 'open_project/authentication/strategies/warden/session'
+
+strategies = [
```

Comment:
It used to be more than 1 value per key. But yeah, I'll go ahead and make it a dictionary.